README.md

```diff
@@ -46,6 +46,11 @@ PDFs are generated inside PDF folder from webpack bundle (dist folder)
 The following plugins are enabled: Markdown, Highlight, Zoom, Notes and Math.
 Refer to [Reveal's documentation](https://revealjs.com/plugins/#built-in-plugins) for usage.
 
+
+## Change slides locale
+
+You can add `--env.lang=fr` when starting sensei to switch to french locale (currently, it only changes the punctation behavior)
```

Comment:
There will be an issue with the docker image as it will override the `CMD` and so we need to explicitly add the `serve` when adding a parameter...

Reply:
Yeah I just rebase it... But I think we need to have a better handling of params now... I will close this PR.